gh-102956: Fix returning of empty byte strings after seek in zipfile … (#103565)
python/cpython: Misc/NEWS.d/next/Library/2023-04-15-14-45-21.gh-issue-102956.Z6qeUy.rst

> Fixed returning of empty byte strings after seek in zipfile module(This wasn't in a release version so the news is probably unnecessary)

Reviewer: Looks like this has now been included in a released version (also, we generally use the imperative mood for NEWS entries -- "fix" rather than "fixed"):

```suggestion
Fix returning of empty byte strings after seek in zipfile module
```
